Tests/notification-extension-tests/NotificationExtensionTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  NotificationExtensionTests.swift
+//  notification-extension-tests
+//
+//  Created by Tapash Majumder on 7/6/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+
+import XCTest
+
+@testable import IterableAppExtensions
+
+class NotificationExtensionTests: XCTestCase {
```

Comment:
Do we need this file?

Reply:
I don't think we need this. If we don't have this file then the coverage for project is reduced. I will rename this so that there is no confusion.